src/style.css

```diff
@@ -0,0 +1,123 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  margin: 0;
+  width: 200px;
+  padding: 30px 14px 15px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  width: 160px;
+  height: 134px;
+  margin: auto;
+}
+
+.card_text--name {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+
+  margin: 45px 0 5px;
+  color: #000;
+}
+
+.card_text--code {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+
+  margin: 0;
+  margin-bottom: 15px;
+  color: #000;
+}
+
+.card__stars {
+  display: flex;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.card__stars__item {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-image: url("./images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.star--active {
+  background-image: url("./images/star-active.svg");
+}
+
+.card__stars__item--text {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 11px;
+  line-height: 13px;
+  margin: 0;
+  margin-left: 17px;
+  color: #000;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.card__price--name {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+  margin: 0;
+  margin-bottom: 1px;
+}
+
+.card__price--number {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+  margin: 0;
+}
+
+.card__button {
+  width: 170px;
+  height: 40px;
+  line-height: 40px;
+  text-align: center;
+  background-color: #00acdc;
+  border-radius: 5px;
+  color: white;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+}
+
+.card__button:hover {
+  background: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
```

Comment:
at this moment when you hovered on your link - border appears and adds 1px, because of box-sizing: border-box; width and height of your button is recalculating, and text could change his position, so its better if you write border property inside the button.